README.md

```diff
@@ -130,6 +130,7 @@ This may contain few sections:
     - `project_path` - path to project source directory
     - `workspace` - PTS workspace path to be used
     - `board` - IUT used. Currently nrf52 is supported only
+    - `board_id` - Segger ID of the IUT used.
```

Comment:
Is this the serial number of the device?

Reply:
Yes I will clarify the naming. "Serial number of the IUT used" more proper?